examples/server.rs

```diff
@@ -216,8 +216,27 @@ fn main() {
             debug!("{} processed {} bytes", conn.trace_id(), read);
 
             let streams: Vec<u64> = conn.readable().collect();
-            for s in streams {
-                handle_stream(conn, s, args.get_str("--root"));
+            'recv: for s in streams {
+                let (read, fin) = match conn.stream_recv(s, &mut buf) {
```

Comment:
Same re: reading data in a loop.

Reply:
I don't think we need to. Right now `stream_recv()` only returns `Done` as error. In the future it should return `InvalidStreamState` if the application tries to read from a local uni stream, but that can be prevented on the application side as well.

It's probably best to avoid the panic on error anyway.